challenge-react/db.json

```diff
@@ -3,31 +3,31 @@
     {
       "id": 1,
       "name": "Baan Kru Noi",
-      "image": "baan-kru-noi.jpg",
+      "image": "http://localhost:3000/images/baan-kru-noi.jpg",
```

Comment:
You shouldn't change the URL of the image in the database.
If you deploy this application to the server. How do you deal with this?